We use the type uintptr_t but we don't directly include the header
file <stdint.h> from which it comes.

--- a/source/include/platform/aclinux.h
+++ b/source/include/platform/aclinux.h
@@ -312,6 +312,7 @@
 #ifdef ACPI_USE_STANDARD_HEADERS
 #include <stddef.h>
 #include <unistd.h>
+#include <stdint.h>
 
 #define ACPI_OFFSET(d, f)   offsetof(d, f)
 #endif
